frontend/packages/dev-console/src/components/pipelineruns/logs/LogSnippetBlock.tsx

```diff
@@ -1,19 +1,30 @@
 import * as React from 'react';
+import { K8sResourceKind } from '@console/internal/module/k8s';
 import { PipelineRun } from '../../../utils/pipeline-augment';
 import LogSnippetFromPod from './LogSnippetFromPod';
 import { PipelineRunErrorDetails } from './pipelineRunLogSnippet';
 
 type LogSnippetBlockProps = {
   children: (logSnippet: string) => React.ReactNode;
   logDetails: PipelineRunErrorDetails;
-  pipelineRun: PipelineRun;
+  pipelineRun?: PipelineRun;
+  taskRun?: K8sResourceKind;
```

Comment:
Maybe we just simplify this to a single `K8sResourceKind` or even better down to just `namespace`?

Reply:
simplified to namespace for all cases